src/Tizen.NUI/src/public/Common/NUIConstants.cs

```diff
@@ -767,10 +767,9 @@ public enum WindowType
         /// <summary>
         /// Used for IME window that is used for keyboard window.
         /// It should be set in Application's New input param when application is created.
-        /// In addition, it is only for internal keyboard application.
-        /// This should be hidden.
+        /// Because IME window type can not change in runtime.
+        /// So, It does not work with Window.Type.
```

Comment:
in `ReserveSizePerOrientationForIME(...)` method, it is IME, but here it is `Ime`. as I know the DOTNET recommends to use pascal case for the names which have more than 3 characters. (https://docs.microsoft.com/ko-kr/visualstudio/code-quality/ca1709?view=vs-2019). but for this case, `IME` looks better. How do you think about?

Reply:
I see. I will update the modified with Ime. Thank you for kind review.